spec/javascripts/unit/AnalyticsSpec.js

```diff
@@ -215,125 +215,4 @@ describe("GOVUK.Analytics", function () {
-      setupQueryString('?status=closed&status=live');
-
-      expect(window.ga.calls.all()[1].args).toEqual(['set', 'dimension24', 'closed|live']);
-    });
-
-    it('should send the status filters as a custom dimension if multiple and in wrong order', function() {
-      setupQueryString('?status=live&status=closed');
-
-      expect(window.ga.calls.all()[1].args).toEqual(['set', 'dimension24', 'closed|live']);
-    });
-
-    it('should send the filter groups as a custom dimension if one', function() {
-      setupQueryString('?status=closed');
-
-      expect(window.ga.calls.all()[2].args).toEqual(['set', 'dimension22', 'status']);
-    });
-
-    it('should send the filter groups as a custom dimension if multiple', function() {
-      setupQueryString('?status=closed&lot=digital-specialists');
-
-      expect(window.ga.calls.all()[3].args).toEqual(['set', 'dimension22', 'lot|status']);
-    });
-
-    it('should send the filter groups as a custom dimension if multiple and wrong order', function() {
-      setupQueryString('?lot=digital-specialists&status=closed');
-
-      expect(window.ga.calls.all()[3].args).toEqual(['set', 'dimension22', 'lot|status']);
-    });
-
-    it('should send the correct custom dimension if all filters are used', function() {
-      setupQueryString('?lot=digital-specialists&status=closed&lot=digital-outcomes&status=live&lot=user-research-participants');
-
-      expect(window.ga.calls.first().args).toEqual(['set', 'dimension21', '32']);
-      expect(window.ga.calls.all()[1].args).toEqual(['set', 'dimension24', 'closed|live']);
-      expect(window.ga.calls.all()[2].args).toEqual(['set', 'dimension23', 'digital-outcomes|digital-specialists|user-research-participants']);
-      expect(window.ga.calls.all()[3].args).toEqual(['set', 'dimension22', 'lot|status']);
-    });
-
-  });
-
-  describe("Opportunity page", function() {
-    var lotData = $('<span data-lot="test-lot"></span>');
-
-    beforeEach(function () {
-      $(document.body).append(lotData);
-    });
-
-    afterEach(function () {
-      lotData.remove();
-    });
-
-    it('should send the lot as a custom dimension for any DOS lot', function() {
-      spyOn(GOVUK.GDM.analytics.location, "pathname")
-        .and
-        .returnValue('/digital-outcomes-and-specialists-3/opportunities/100');
-      window.GOVUK.GDM.analytics.pageViews.init();
-      expect(window.ga.calls.all().map(function (i) {return i.args})).toContain(['set', 'dimension26', 'test-lot']);
-    });
-
-    it('should not send the lot as a custom dimension for any non DOS lot', function() {
-      spyOn(GOVUK.GDM.analytics.location, "pathname")
-        .and
-        .returnValue('/not-digital-outcomes-and-specialists/opportunities/100');
-      window.GOVUK.GDM.analytics.pageViews.init();
-      expect(window.ga.calls.all().map(function (i) {return i.args})).not.toContain(['set', 'dimension26', 'test-lot']);
-    });
-  });
```

Comment:
Though this is more a test of the code in the toolkit now, is it worth still keeping this in here as an integration test of sorts - to ensure the toolkit code is being used right?

If it would mean lots of duplicated code remaining in the various frontend repos, the common body of the test could be moved to the toolkit...

Reply:
In theory yes, however these specific tests aren't actually testing the Briefs FE! It's all public DOS pages and searches, copied over boilerplate from the Buyer FE.

Reply:
God damn that's weird